### DISCUSSION TITLE
Question about gtk-4.0 theme instructions during latest Manjaro upgrade

### DISCUSSION THREAD

**Discussion by heapifyman** [Manjaro-Sway/manjaro-sway#526] [Q&A] [answered]
While upgrading to the [latest Manjaro version](https://forum.manjaro.org/t/stable-update-2023-06-04-kernels-gnome-44-1-plasma-5-27-5-python-3-11-toolchain-firefox/141610) I noticed this message in the log:

```
(257/385) upgrading matcha-gtk-theme [--------------------------------------------------------] 100%

To force Libadwaita appliations to use the gtk-4.0 theme,
symlink the gtk-4.0 theme to config folder
(replace Matcha-dark-sea with chosen theme variant):

mkdir -p ~/.config/gtk-4.0/
ln -sf /usr/share/themes/Matcha-dark-sea/gtk-4.0/{assets,gtk.css,gtk-dark.css} ~/.config/gtk-4.0/
```

Would that be relevant for manjaro-sway? Or can it be safely ignored?

**Comment by boredland** [comment]
Hm, good catch. I haven't dealt with it all long, that I am not even sure how exactly we configure the GTK theme to be used... Relatively sure not via that dir.

**Comment by boredland** [comment, marked as answer]
I think this is solved with a custom script updating the gtk4 theme.
